src/FixedDataTable.js

```diff
@@ -508,43 +508,77 @@ class FixedDataTable extends React.Component {
     this._didScrollStop = debounceCore(this._didScrollStopSync, 200, this);
     this._onKeyDown = this._onKeyDown.bind(this);
 
-    this._wheelHandler = new ReactWheelHandler(
-      this._onScroll,
-      this._shouldHandleWheelX,
-      this._shouldHandleWheelY,
-      this.props.isRTL,
-      this.props.stopScrollDefaultHandling,
-      this.props.stopScrollPropagation
-    );
-
-    this._touchHandler = new ReactTouchHandler(
-      this._onScroll,
-      this._shouldHandleTouchX,
-      this._shouldHandleTouchY,
-      this.props.stopScrollDefaultHandling,
-      this.props.stopScrollPropagation
-    );
+    this._setupHandlers();
```

Comment:
The registration of the scroll/touch handlers have been moved to a new helper `_setupHandlers()`.
See https://github.com/schrodinger/fixed-data-table-2/pull/657 for more details.